recipes/hyperion/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+python setup.py install
+./configure --prefix=$PREFIX
+make
+make install
```

Comment:
No, but I've included tests to make sure the executables exist in the test section. I was going to include a small test file to run them all to make sure they run (the test would run in a few seconds)